frontend/src/metabase-lib/order_by.unit.spec.ts

```diff
@@ -1,40 +1,59 @@
-import type { Field } from "metabase-types/api";
-import { createQuery, SAMPLE_DATABASE } from "./test-helpers";
+import { createQuery } from "./test-helpers";
 import * as ML from "./v2";
 
+// This is a convenience for finding an orderable column (as an opaque object) by name
+const findOrderableColumn = (
+  query: ML.Query,
+  tableName: string,
+  fieldName: string,
+): ML.ColumnMetadata => {
+  const columns = ML.orderableColumns(query);
+  const column = columns?.find((column: ML.ColumnMetadata) => {
+    const displayInfo = ML.displayInfo(query, column);
+    return (
+      displayInfo?.table?.name === tableName && displayInfo?.name === fieldName
+    );
+  });
```

Comment:
I think `ML.orderableColumns` is guaranteed to return an array, so `columns?.find` seems redundant. Also, I believe we can avoid explicitly typing the `column` inside `find's` callback — TypeScript should be smart enough to pick it up from `orderableColumns` return type